checks.d/http_check.py

```diff
@@ -243,7 +244,8 @@ def _check(self, instance):
                 self.log.debug("Weak Ciphers will be used for {0}. Suppoted Cipherlist: {1}".format(
                     base_addr, WeakCiphersHTTPSConnection.SUPPORTED_CIPHERS))
 
-            r = sess.request('GET', addr, auth=auth, timeout=timeout, headers=headers, proxies=instance_proxy,
+            r = sess.request('GET', addr, auth=auth, timeout=timeout, headers=headers,
+                             proxies=instance_proxy, allow_redirects=allow_redirects,
```

Comment:
In `requests` `allow_redirects` defaults to `True` as well so 👍  for the the default value as it maintains backward compatibility.